### PR TITLE
Alerts

### PR DESCRIPTION
**Pull request by optimumtact**
WIP rework of the alert code

TODO, alarmlistener should obviously be a datum component, along with the alarm datum itself

maybe it will benefit from a subsystem, or I put it on SSOBJ

### CODE REVIEW

**Comment by Qustinnus** [comment]
**feature freeze!!!!!!!!**

**Comment by duncathan** [comment]
foolish mortal

**Comment by optimumtact** [comment]
yes this has been in progress for so long datum components were literally added while Iwas doing it

**Comment by optimumtact** [comment]
ps maintainers still merge that other pr this will take me a bit to finish as i have to rework the computer/AI code

AI code specifically put me off finishing for quite some time

File: tgstation.dme [hunk 1]
```diff
@@ -118,6 +119,8 @@
 #include "code\_globalvars\genetics.dm"
 #include "code\_globalvars\logging.dm"
 #include "code\_globalvars\misc.dm"
+#include "code\_globalvars\sensitive.dm"
```
Comment (optimumtact) [inline tgstation.dme]: was this removed

Comment (ninjanomnom) [reply tgstation.dme]: looks like it

Comment (Cyberboss) [reply tgstation.dme]: It was replaced with config datums

**Comment by ninjanomnom** [comment]
@optimumtact The day comes when you are the one getting the stale tag

**Pull request closed without merging**